infra/base-images/base-runner/test_all.py

```diff
@@ -82,7 +82,8 @@ def find_fuzz_targets(directory, fuzzing_language):
       continue
     if not os.stat(path).st_mode & EXECUTABLE:
       continue
-    if fuzzing_language != 'python' and not is_elf(path):
+    if (fuzzing_language != 'python' and fuzzing_language != 'jvm' and
```

Comment:
Can you comment here that we dont want to do bad_build_check on fuzz targets built with jazzer?
Also, why don't we want to do the checks?

Reply:
The intention behind this particular change is to not skip Jazzer fuzz targets just because they are not binaries (they are shell scripts, just like for Python). Jazzer fuzz targets are still subject to most the `bad_build_check` checks, just not the ones that apply `objdump` to a binary target.

Reply:
OK, sorry for the confusion (here and in the other comment).
I thought the purpose of this change was to disable bad_build_check from being run on jazzer targets.
If we were disabling it, I didn't know why you were changing bad_build_check for jazzer if it wouldn't be used with jazzer.
Could you just add a comment here that fuzz targets are elfs unless we are fuzzing python or java?
Thanks!

Reply:
I added the comment.